fluss-client/src/main/java/com/alibaba/fluss/client/table/scanner/log/LogFetchCollector.java

```diff
@@ -263,8 +263,9 @@ private void handleInitializeErrors(
             metadataUpdater.checkAndUpdateMetadata(tablePath, tb);
         } else if (error == Errors.LOG_OFFSET_OUT_OF_RANGE_EXCEPTION) {
             throw new FetchException(
-                    String.format("The fetching offset %s is out of range", fetchOffset),
-                    error.exception());
+                    String.format(
+                            "The fetching offset %s is out of range: %s",
+                            fetchOffset, errorMessage));
```

Comment:
Sorry, I couldn't understand this change. It seems the new version have less error information (strip out the exception stack). The `error.exception()` should contain information in `errorMessage`.

Reply:
For common error, The `error.exception()` indeed contain information in `errorMessage`. But in method `LogFetchCollector#initialize()`, it refactored `ApiError` into `Errors` and `ApiMessage.messageWithFallback()` to pass to the `handleInitializeErrors` method. As a result, the `Errors` class no longer contains detailed error messages, and debugging output:

![image](https://github.com/user-attachments/assets/b19afc5e-d9be-4418-9063-1ef32734e8c6)

Reply:
OK. Then I suggest to use `error.exception(errorMessage)` to carry the API exception in the error stack.